src/epub/text/chapter-56.xhtml

```diff
@@ -154,7 +155,7 @@
 			<p>And they parted. Petronius returned to his cubiculum; but Vinicius went to look from a distance at the prison, and thence betook himself to the slope of the Vatican hill⁠—to that hut of the quarryman where he had received baptism from the hands of the Apostle. It seemed to him that Christ would hear him more readily there than in any other place; so when he found it, he threw himself on the ground and exerted all the strength of his suffering soul in prayer for mercy, and so forgot himself that he remembered not where he was or what he was doing. In the afternoon he was roused by the sound of trumpets which came from the direction of Nero’s Circus. He went out of the hut, and gazed around with eyes which were as if just opened from sleep.</p>
 			<p>It was hot; the stillness was broken at intervals by the sound of brass and continually by the ceaseless noise of grasshoppers. The air had become sultry, the sky was still clear over the city, but near the Sabine Hills dark clouds were gathering at the edge of the horizon.</p>
 			<p>Vinicius went home. Petronius was waiting for him in the atrium.</p>
-			<p>“I have been on the Palatine,” said he. “I showed myself there purposely, and even sat down at dice. There is a feast at the house of Vinicius this evening; I promised to go, but only after midnight, saying that I must sleep before that hour. In fact I shall be there, and it would be well wert thou to go also.”</p>
+			<p>“I have been on the Palatine,” said he. “I showed myself there purposely, and even sat down at dice. There is a feast at the house of Anicius this evening; I promised to go, but only after midnight, saying that I must sleep before that hour. In fact I shall be there, and it would be well wert thou to go also.”</p>
```

Comment:
Yup, here's the scan: https://babel.hathitrust.org/cgi/pt?id=njp.32101074898303&view=1up&seq=287&q1=house%20of%20anicius

I was very confused when I was reading this for the first time, since Petronius was talking to Vinicius--did he not know that there was supposed to be a feast in his own house? 🤔 So I checked the scans and found this error.